packages/cyberstorm/src/components/componentStyles/PackageCard.module.css

```diff
@@ -0,0 +1,130 @@
+/* Common PackageCard styles */
+.root {
+  display: flex;
+  flex: none;
+  flex-direction: column;
+  gap: var(--gap-m);
+  align-items: flex-start;
+
+  border-radius: var(--border-radius-m);
+  overflow: hidden;
+}
+
+.root:hover {
+  box-shadow: var(--box-shadow-1);
+}
+
+.imageWrapper {
+  position: relative;
+  width: 100%;
+  padding-bottom: 100%;
+  overflow: hidden;
+  transition: 0.2s;
+}
+
+.root:hover .imageWrapper {
+  filter: brightness(1.15);
+}
+
+/* Styles for PackageCard's image */
+.image {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: 100%;
+  height: auto;
+  transform: translate(-50%, -50%);
+  transition: 0.2s;
+}
+
+.root:hover .image {
+  transform: translate(-50%, -50%) scale(1.05);
+}
+
+.content {
+  display: flex;
+  flex: none;
+  flex-direction: column;
+  gap: var(--gap-s);
+  padding: 0 var(--padding-m);
+}
+
+/* Styles for PackageCard's Package name (title) */
+.name {
+  color: var(--color-text--default);
+  font: var(--font-sora);
+  font-weight: var(--font-weight-bold);
+  line-height: var(--text-xl);
+}
+
+/* Styles for PackageCard's Author */
+.author {
+  font: var(--font-inter);
+  font-size: var(--text-xs);
+}
+
+.author_label {
+  color: var(--color-cyber-green);
+  font-weight: var(--font-weight-bold);
+  text-decoration: none;
+}
+
+.author_prefix {
+  color: var(--color-text--tertiary);
+}
+
+/* Styles for PackageCard's description */
+.description {
+  display: -webkit-box;
+  overflow: hidden;
+  color: var(--color-text--secondary);
+  font-size: var(--text-s);
+  font-family: Inter, sans-serif;
```

Comment:
This and other instances of hard coded font-families should use a variable. Also while at it, it would be better to create variables which use `--font-inter` and `--font-sora` instead, e.g. `--font--body` and `--font--heading` respectively. This way we can change the fonts in one place without having to go through every component.

Reply:
👍